frontend/src/app.css

```diff
@@ -43,3 +43,14 @@
     columns: 5rem auto;
   }
 }
+
+@variants {
+  .filters-content{
+    height: calc(100% - 110px);
+  }
+  @media (min-width: 768px) {
+    .filters-content{
+      height: 100%;
+    }
+  }
```

Comment:
I believe you intended to use `@screen` directive here, see https://tailwindcss.com/docs/functions-and-directives#screen.

However I think that we can just apply `md:h-full` to the element without this CSS code.